analyzers/tests/SonarAnalyzer.UnitTest/ControlFlowGraph/ControlFlowGraphTest.cs

```diff
@@ -5219,6 +5219,40 @@ public void Cfg_New_TargetTyped()
 
         #endregion
 
+        #region "Tuples"
+
+        [TestMethod]
+        [TestCategory("CFG")]
+        public void Cfg_Tuple_Create()
```

Comment:
it would also be nice to have a test with expressions in the tuple creation, e.g.
`var (a,b) = (Foo(), 1 + Bar());`